src/backend/routers/conversation.py

```diff
@@ -45,15 +39,18 @@
     filter_conversations,
     generate_conversation_title,
     get_documents_to_rerank,
+    getMessagesWithFiles,
     validate_conversation,
 )
 from backend.services.file import (
-    get_file_content,
+    FileService,
+    attach_conversation_id_to_files,
     validate_batch_file_size,
     validate_file,
     validate_file_size,
 )
 
+file_service = FileService()
```

Comment:
Suggestion: since you're going with a singleton-like implementation, rather than define the file service globally in the file that uses it, in the services/file.py can you define some method to get it?

Something like:

```
file_service = None

class FileService:
    ...

def get_file_service():
     if not file_service:
            file_service = FileService()
     
     return file_service
```

Then just import `get_file_service`, and wherever you need to use it, do 
`get_file_service().mymethod()`

Reply:
Ping me if you have questions about this

Reply:
yep that makes so much more sense. thanks!